packages/studio/package.json

```diff
@@ -20,7 +20,7 @@
     "@minoru/react-dnd-treeview": "^3.4.1",
     "@restart/ui": "^1.5.2",
     "@vitejs/plugin-react": "^4.0.4",
-    "@yext/studio-plugin": "*",
+    "@yext/studio-plugin": "0.19.0",
```

Comment:
out of curiosity now that the versions are 0.20.0, if we forget to update it here and keep it at 0.19.0 what happens?

Reply:
The symlink is still preserved in the package-lock, but the test-site will still be using the old version (which in this case caused an error since 0.19.0 is missing the TailwindPropInput stuff). This because the test site only has `studio` as a dependency, which still has the old `studio-plugin` version as a dependency.